src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -188,5 +189,24 @@ public override string ToString()
         {
             return DisplayText;
         }
+
+        private List<string> tags = null;
+        public IEnumerable<string> Tags
```

Comment:
At the risk of sounding like a broken record, I thought this was what we agreed upon at lunch :smile: